pkg/ansisql/checks.go

```diff
@@ -187,28 +186,26 @@ func (c *NegativeCheck) Check(ctx context.Context, ti *scheduler.ColumnCheckInst
 }
 
 type CustomCheck struct {
-	conn      connectionFetcher
-	extractor query.QueryExtractor
+	conn     connectionFetcher
+	renderer jinja.RendererInterface
 }
 
 func NewCustomCheck(conn connectionFetcher) *CustomCheck {
-	queryExtractor := query.WholeFileExtractor{
-		Fs:       afero.NewOsFs(),
-		Renderer: jinja.NewRendererWithYesterday("your-pipeline-name", "your-run-id"),
-	}
-	return &CustomCheck{conn: conn, extractor: &queryExtractor}
+	// TODO: this needs to use an actual renderer instead of the yesterday, since this `NewRendererWithYesterday` does not honor
+	// the parameters passed to the `bruin run` command.
+	return &CustomCheck{conn: conn, renderer: jinja.NewRendererWithYesterday("your-pipeline-name", "your-run-id")}
```

Comment:
@turtleDev seems like you used this `NewCustomCheck` constructor in the EMR Serverless checks, but this seems to have been always using a jinja renderer that's not really accurate. this constructor seems to be only used in tests, you might need to refactor your implementation to use the correct renderer coming from above.